guide/payments/request.md

```diff
@@ -0,0 +1,471 @@
+   layout = "shadow"
+   caption = "Single-use payment requests, like invoices, are useful for in-person payments."
+   alt-text = "Screen showing single-use invoice."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/my-contact-card.png"
+   retina = "/assets/images/guide/payments/request/my-contact-card@2x.png"
+   layout = "shadow"
+   caption = "Reusable payment requests, like Lightning addresses, are useful for receiving recurring payments."
+   alt-text = "Screen showing reusable Lightning address."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/withdraw-request.png"
+   retina = "/assets/images/guide/payments/request/withdraw-request@2x.png"
+   layout = "shadow"
+   caption = "A withdraw request is another way a user can request bitcoin."
+   alt-text = "Screen showing withdraw request UI."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+## Single-use payment requests
+
+The most common way bitcoin is requested today is in the form of single-use payment requests. For Lightning, a standard [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoice is used. For on-chain, [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}) are used. Addresses can be reused though this is not advised due to privacy concerns. single-use invoices and addresses need to be easily and conveniently generated within a wallet application.
+
+### Amount
+
+When generating a single-use payment request, users need to be able to add an amount to be paid.  Entering an amount from the home screen can speed up the requesting flow. Alternatively, users can add an amount to an already generated payment request. Entering an amount needs to be clear to prevent users entering incorrect amounts.
+
+Amount units need to be toggleable between bitcoin, satoshi and local currency values as senders may request in a set unit. Ensure the unit being entered is prominent to prevent entering of the wrong unit amount. More on the [Units & Symbols page]({{ "/guide/payments/units-and-symbols/" | relative_url }}).
+
+### Address types
+
+There are several address types that can be used for on-chain transfer requests. Each address type has unique benefits, such as SegWit saving on network fees. Supporting every address type makes a wallet more interoperable and lets users access the benefits offered by each type. More on [address types]({{ "/guide/glossary/address/" | relative_url }}) page.
+
+### Meta data
+
+A payment request with only standard data like an amount and date communicates little to both parties about what the payment was for. Users should have the option to attach a note for record keeping and/or for the sender to read. Adding tags, labels and any other metadata that may be relevant to give more context to the payment is also beneficial.
```

Comment:
Simplifying the first sentence a bit. Also removed `and/or, personally it kills the readability for me, but if you like it, feel free to keep it.

```suggestion
A payment request with only standard data like an amount and date communicates little to both parties about the payment. Users should have the option to attach a note for record-keeping and for the sender to read. Adding tags, labels and any other metadata that may be relevant to give more context to the payment is also beneficial.
```